contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3886,6 +3980,106 @@ call_prev_ProcessUtility(PlannedStmt *pstmt,
 								queryEnv, dest, qc);
 }
 
+/*
+ * Get the oid and acl of a TSQL proc by name. Raises an error if the proc doesn't exist, or if there are
+ * multiple procs with the same name (and different parameters). Also sets isSameFunc based on whether
+ * the found proc is the exact same proc as requested (i.e. the parameters match).
+ */
+static void 
+pltsql_proc_get_oid_proname_proacl(AlterFunctionStmt *stmt, ParseState *pstate, Oid *oid, Acl **acl, bool *isSameFunc)
```

Comment:
For the **acl, I didn't see anything done in this function.

Reply:
We set the acl pointer in this function so we can copy it into the new procedure in pg_proc_update_oid_acl